### ISSUE TITLE
Reproducibility: the tree_search generates too small tree

### ISSUE THREAD

**Issue by KexinFeng** [Infini-AI-Lab/Sequoia#13]
Hi, 

I was trying to reproduce the numbers in the paper, but with the `demo-config.json`, plus the acceptance vector in the repo or the acceptance vector I tested myself, the generated trees are all very small and somewaht fixed: 
```
0 _ 1 _ 3
 \_ 2

or

0 _ 1 _2 _3
```

But on the other hand, the growmaps in the two folders are generally very large, typically of size 128, 64, 32. Do you know what the possible reason is that the tree I generated is small and how to reproduce the growmaps in those two folders?

Thank you!

**Comment by dreaming-panda** [comment]
Can you send me your demo-config.json?  
PS: the original demo-config.json is just a demo. You need to modify the content to generate a tree you want.

**Comment by dreaming-panda** [comment]
{
    "acceptance_rate_vector": "acceptance-rate-vector.pt",
    "max_depth": 15,
    "max_budget": 128,
    "draft_time": 0.0003,
    "valid_budget": [1, 2, 4, 8, 16, 32, 64, 128],
    "target_time":[0.025, 0.025, 0.025, 0.025, 0.025, 0.027, 0.030, 0.035],
    "dst": "demo_tree.pt"
}

p = [0.0000, 0.4803, 0.1104, 0.0576, 0.0373, 0.0265, 0.0211, 0.0170, 0.0135,
        0.0113, 0.0093, 0.0087, 0.0075, 0.0067, 0.0058, 0.0061, 0.0049] 
might be a proper example to generate a tree of size of 32. 

Some explanation:   draft_time is the time for one draft model's forward pass.  target_time is the time for one draft model's forward pass corresponding to the valid budget.

**Comment by dreaming-panda** [comment]
Yes, the generated tree sizes can only be numbers from "valid_budget".

**Comment by singularity-s0** [comment]
How to determine the optimal depth and budget? Can you share the `config.json` that generated the example A100 and L40 growmaps in this repository?

**Comment by singularity-s0** [comment]
> Some explanation: draft_time is the time for one draft model's forward pass. target_time is the time for one draft model's forward pass corresponding to the valid budget.

Is there a typo here? Shouldn't `target_time` be the time for a verification pass?

**Comment by briskerkazoos** [comment]
> > Some explanation: draft_time is the time for one draft model's forward pass. target_time is the time for one draft model's forward pass corresponding to the valid budget.
> 
> Is there a typo here? Shouldn't `target_time` be the time for a verification pass?

I have the same question.

> "draft_time": 0.0003

What is the unit of this value? I don't think any model can complete the forward pass within 0.0003 seconds?
